pkgs/tools/misc/osm2pgsql/default.nix

```diff
@@ -12,33 +13,42 @@
 , luajit
 , libosmium
 , protozero
+, rapidjson
 }:
 
 stdenv.mkDerivation rec {
   pname = "osm2pgsql";
-  version = "1.6.0";
+  version = "1.7.0";
 
   src = fetchFromGitHub {
     owner = "openstreetmap";
     repo = pname;
     rev = version;
-    sha256 = "sha256-6FVMv+DowMYdRdsQFL2iwG/V9D2cLWkHUVkmR3/TuUI=";
+    hash = "sha256-MWJzCZdqvy/nH1Doj0fmGuzTubaHDnPOED7qgzvJ3ZU=";
   };
 
+  postPatch = ''
+    rm -r contrib
```

Comment:
Why is this needed?

Reply:
It removes bundled libraries (libosmium, protozero, fmt, rapidjson), just to make sure external ones are used.

Reply:
Cool. A comment to this effect would be really helpful.